composeApp/src/desktopMain/kotlin/org/ooni/probe/BuildDependencies.kt

```diff
@@ -20,14 +21,13 @@ import org.ooni.probe.di.Dependencies
 import org.ooni.probe.shared.Platform
 import org.ooni.probe.shared.PlatformInfo
 
-// TODO: Desktop - proper directories for each system
-private const val BASE_FILES_DIR = "../temp"
+private val projectDirectories = ProjectDirectories.from(null, null, "OONI Probe")
```

Comment:
I think we should be ussing the package name here.

Reply:
I'll go with the full set of arguments then:

<img width="634" alt="Screenshot 2025-03-31 at 11 03 38" src="https://github.com/user-attachments/assets/22f90fe3-dda8-41be-9dab-e408197fc10d" />